README.md

```diff
@@ -1,3 +1,39 @@
-# coSMicQC
+<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">
 
-coSMicQC is a software tool for single-cell quality control of morphology datasets.
+# Single cell Morphology Quality Control
+
+[![Build Status](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml/badge.svg?branch=main)](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml?query=branch%3Amain)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
```

Comment:
Consider adding another line for separating the text from the badges to help increase legibility. I'm not sure what'd be best here, but felt that it might help! Feel free to leave things as is with the spacing if you don't feel the same.

Reply:
It looks like with the addition of poetry and pypi version that I have added another line at the bottom to separate the text, so I think I have implemented this. Thanks for the suggestion!